app/css/views/themes.css

```diff
@@ -42,4 +42,5 @@
   margin: 0 0 0 1rem;
   padding: 0;
   right: 0;
+  font-size: 0;
```

Comment:
__question:__ why do we need that and ```x``` as button text (what we really need there is ```title``` btw)?